plottr/data/datadict.py

```diff
@@ -174,7 +175,7 @@ def meta_items(self, data: Union[str, None] = None,
                         n = k
                     yield n, v
 
-    def data_vals(self, key: str) -> Union[Sequence, np.ndarray]:
+    def data_vals(self, key: str) -> np.ndarray:
```

Comment:
this means that datadicts should always be made from dicts of numpy arrays, right? iguess, it's true within existing plottr codebase?

Reply:
Indeed. All the code is typed and this does not trigger any type errors. And there was no handling of Sequences to be found in the codebase